File: src/io/extensionToMeshIO.ts
const extensionToIO = new Map([
  ['vtk', 'VTKPolyDataMeshIO'],
  ['VTK', 'VTKPolyDataMeshIO'],
  ['byu', 'BYUMeshIO'],
  ['BYU', 'BYUMeshIO'],
  ['fsa', 'FreeSurferAsciiMeshIO'],
  ['FSA', 'FreeSurferAsciiMeshIO'],
  ['fsb', 'FreeSurferBinaryMeshIO'],
  ['FSB', 'FreeSurferBinaryMeshIO'],
  ['obj', 'OBJMeshIO'],
  ['OBJ', 'OBJMeshIO'],
  ['off', 'OFFMeshIO'],
  ['OFF', 'OFFMeshIO'],
  ['stl', 'STLMeshIO'],
  ['STL', 'STLMeshIO'],
  ['swc', 'SWCMeshIO'],
  ['SWC', 'SWCMeshIO'],
  ['iwm', 'WasmMeshIO'],
  ['iwm.cbor', 'WasmMeshIO'],
  ['iwm.cbor.zstd', 'WasmZstdMeshIO']
])

export default extensionToIO

File: packages/mesh-io/python/itkwasm-mesh-io-wasi/itkwasm_mesh_io_wasi/extension_to_mesh_io.py
from collections import OrderedDict

extension_to_mesh_io = OrderedDict([
    ('.vtk', 'vtk_poly_data'),
    ('.byu', 'byu'),
    ('.fsa', 'free_surfer_ascii'),
    ('.fsb', 'free_surfer_binary'),
    ('.obj', 'obj'),
    ('.off', 'off'),
    ('.stl', 'stl'),
    ('.swc', 'swc'),
    ('.iwm', 'wasm'),
    ('.iwm.cbor', 'wasm'),
    ('.iwm.cbor.zst', 'wasm_zstd'),
])
